force-app/main/default/classes/DMLOperations.cls

```diff
@@ -213,7 +298,14 @@ public with sharing class DMLOperations {
      */
     public static void createAndDeleteCases(String accountId, Integer numOfCases) {
         // Insert the cases
-
+        List<Case> casesToInsert = new List<Case>();
+        Case myCase = new Case();
+        for(Integer i = 0; i < numOfCases ; i++){
+            myCase.AccountId = accountId;
+        }
+        insert casesToInsert;
```

Comment:
In the end, what are you actually inserting and deleting? Are there any cases in `casesToInsert`? According to your code, this list is always empty. Check your code to see what's going wrong.

Unfortunately, the test can only check to see that no cases exist. That's why your test passed. Be sure to redo this method and make sure you're creating multiple cases. (Hints: Line 302 should be inside the loop and you need to add each case to the list.)